### PR TITLE
Change default to use Okta-hosted login

### PR DESCRIPTION
**Pull request by mraible**

### CODE REVIEW

**Review by Quorralyne — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Okta Angular + Custom Login Example
+# Okta Angular + Okta Hosted Login Example
```
Comment (alisaduncan) [inline README.md:1]: We should consider changing the name to "Okta Angular + Okta Redirect Model Example". Chris Mills checked with Product and that's the name for OIE and from what I understand, everything is being updated with that naming for consistency.

File: src/app/home/home.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+describe('HomeComponent', () => {
+  let component: HomeComponent;
+  let fixture: ComponentFixture<HomeComponent>;
+  const config = {
+    issuer: 'https://not-real.okta.com',
+    clientId: 'fake-client-id',
+    redirectUri: 'http://localhost:4200'
+  };
+  const oktaAuth = new OktaAuth(config);
```
Comment (alisaduncan) [inline src/app/home/home.component.spec.ts:16]: I'm happy to make a follow up PR for mocks so we can provide a template for testing that devs can run with.

**Review by alisaduncan — APPROVED** [review]
One comment on naming for consistency

File: src/app/home/home.component.spec.ts (hunk 2 above)
Comment (mraible) [reply src/app/home/home.component.spec.ts:16]: That'd be awesome! Please keep in mind you'll want to update both the `main` and `widget` branches.

File: README.md (hunk 1 above)
Comment (alisaduncan) [reply README.md]: I agree, but I'm a little fuzzy about how the official sample/quickstart samples will work with our new updates. I used the name "Redirect Model" in the quickstart repo I created.

https://github.com/oktadev/okta-angular-quickstart

**Pull request merged**